english/eating-walls/find-a-toilet/door/run-in/run-into-room.md

```diff
@@ -0,0 +1,7 @@
+You run into the room and slam the door behind you! You don't know whose hand that was and you don't want to.
```

Comment:
It looks like there is no link to this new file.  Did you forget to commit that change?  Make sure you create (and commit) an option for each new "page" you create.